examples/helm/values.yaml

```diff
@@ -11,6 +11,7 @@ fullnameOverride: ""
 
 enableAuth: false
 enableGuests: true
+# publicURL: ""
```

Comment:
Why not leaving publicURL uncommented ? (As it as a default value)

Reply:
Added